bindings/py/cpp_src/bindings/algorithms/py_SDR.cpp

```diff
@@ -289,33 +289,27 @@ special, it is replaced with the system time.  The default seed is 0.)",
         }));
 
 
-        py::class_<SDR_Proxy, SDR> py_Proxy(m, "SDR_Proxy",
-R"(SDR_Proxy presents a view onto an SDR.
-    * Proxies always have the same value as their source SDR.
-    * Proxies can be used in place of an SDR.
-    * Proxies are read only.
-    * Proxies can have different dimensions than their source SDR.
+        py::class_<SDR_Reshape, SDR> py_Reshape(m, "SDR_Reshape",
+R"(SDR_Reshape presents a view onto an SDR with different dimensions.
+    * The resulting SDR always has the same value as its source SDR.
+    * The resulting SDR is read only.
 
 Example Usage:
     # Convert SDR dimensions from (4 x 4) to (8 x 2)
     A = SDR([ 4, 4 ])
-    B = SDR_Proxy( A, [8, 2])
+    B = SDR_Reshape( A, [8, 2])
```

Comment:
do you want to go with the namespace `nupic::sdr` and drop the SDR_ from name? So it becomes `sdr::Reshape`?

Reply:
Yes, but I'd like to make the name space in its own PR.  Changing name spaces is going to make a big `git diff` and I'd like to do that all in one PR.